sdk/signalr/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/rest.py

```diff
@@ -0,0 +1,969 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+__all__ = [
+    'build_add_connection_to_group_request',
+    'build_add_user_to_group_request',
+    'build_connection_exists_request',
+    'build_group_exists_request',
+    'build_check_permission_request',
+    'build_user_exists_request',
+    'build_user_exists_in_group_request',
+    'build_close_client_connection_request',
+    'build_grant_permission_request',
+    'build_healthapi_get_health_status_request',
+    'build_remove_connection_from_group_request',
+    'build_remove_user_from_all_groups_request',
+    'build_remove_user_from_group_request',
+    'build_revoke_permission_request',
+    'build_send_to_all_request',
+    'build_send_to_connection_request',
+    'build_send_to_group_request',
+    'build_send_to_user_request'
+]
+from typing import TYPE_CHECKING
+from msrest import Serializer
+from azure.core.pipeline.transport._base import _format_url_section
+from azure.messaging.webpubsubservice.core.rest import HttpRequest
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from typing import Any, IO, List, Optional, Union, Dict
+    from typing_extensions import Literal
+    Permissions = Union[Literal['joinLeaveGroup'], Literal['sendToGroup']]
```

Comment:
Not for the generator - I believe `Literal` supports multiple options: `Literal['joinLeaveGroup', 'sendToGroup']`

Reply:
You are correct. Today I learned something new :).